src/i965_output_dri.c

```diff
@@ -35,7 +35,7 @@
 typedef struct dri_drawable *(*dri_get_drawable_func)(
     VADriverContextP ctx, XID drawable);
 typedef union dri_buffer *(*dri_get_rendering_buffer_func)(
-        VADriverContextP ctx, struct dri_drawable *d);
+            VADriverContextP ctx, struct dri_drawable *d);
 typedef void (*dri_swap_buffer_func)(
```

Comment:
@pengche1, It seems the code is indented incorrectly by the tool if the type of the returned value is union.  I will update the code to avoid this case.